### DISCUSSION TITLE
Few updates regarding Task 1

### DISCUSSION THREAD

**Discussion by chulminy** [chulminy/CIVE497-CIVE700#150] [General]
All 

1. For Q3, it's not a question for implementing an "eye detection" algorithm. You can manually pick the coordinates of the the eye of the baby and draw the circles. 
2. For Q4, you are required to develop and implement your own algorithm to adjust the brightness of a photo. This task goes beyond simply increasing or decreasing brightness levels. The objective is to modify the photo's brightness based on its initial state. If the photo is initially dark, your algorithm should enhance its brightness. Conversely, if the photo is overly bright, your algorithm should reduce the brightness to achieve a balanced exposure, similar to the provided example.

Thanks

Chul Min

**Comment by lesani-ali** [comment]
About question 4, do you mean we should not use histogram equalization? Should we use other algorithms? I would appreciate it if you could clarify it more.

Thanks

**Reply by chulminy** [reply]
I would ask the students to think about how to equalize the brightness, rather than using existing function. Is it too hard?

**Comment by noreengao0924** [comment]
I'm also confued about question 4. In the example, it already uses the PIL's equalize function and manual contrast adjustment to adjust image contrast values. What kind of other algorithms are we expected to use? 
~Thanks

**Reply by chulminy** [reply]
Not expected to use. You need tp design your own function to adjust contrast and explain it. See my implementation. That's the one of example.
